Dido's project/about.html

```diff
@@ -0,0 +1,78 @@
+      <header>
+        <div class="H1">
+          <h1>
+            Want to make your CLK look unique? We have what you are looking for.
+          </h1>
+        </div>
+        <div class="H2">
+          <h2>Welcome to Driftworld!</h2>
+        </div>
+
+        <div>
+          <h2 class="Lorem1">Why choose us?</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam,
+            fugit molestias quam iure magni ut quidem omnis deleniti quod ullam.
+            Aperiam similique facilis ab earum culpa et architecto repellendus
+            sunt rem, hic expedita officia voluptatibus necessitatibus est quos
+            delectus, possimus iure fugiat commodi doloremque exercitationem
+            voluptatum iusto, enim tempore? Ipsam laborum exercitationem earum
+            ab deleniti minima consequatur, fugit omnis ipsa facilis ullam
+            doloremque aspernatur totam, dolorum sit magnam! Suscipit iste
+            praesentium et quas exercitationem expedita voluptatibus officia
+            eum, nihil doloremque accusantium iusto quibusdam maxime! Enim
+            impedit eos officiis mollitia soluta. Reiciendis inventore facere
+            voluptates provident dolore quia totam, sequi fugit?
+          </p>
+        </div>
+        <div>
+          <h2 class="Lorem2">How it all began.</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquid
+            quis, hic repudiandae voluptatem laboriosam porro sint ad veniam
+            vero culpa voluptatum modi voluptas repellat quibusdam dolorem
+            praesentium aut. Dicta itaque veniam soluta ex dolorem quas aut vero
+            tempore amet illo modi sint excepturi deleniti, quisquam eveniet,
+            perferendis a cumque dolores.
+          </p>
+        </div>
+        <div>
+          <h2 class="Lorem3">Our mission.</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut
+            recusandae rem quisquam, repudiandae, eos iure nihil sint natus
+            repellat beatae animi, nesciunt voluptate tempora quasi? Laborum
+            eius quo similique voluptates delectus qui possimus et hic quibusdam
+            iure doloremque ea id nostrum fuga, debitis quas! Voluptatibus,
+            culpa? Doloribus voluptas cumque assumenda nostrum, eligendi atque
+            voluptatem sed doloremque omnis fuga unde? Ea voluptatem enim
+            assumenda magni rerum deleniti maiores dolor debitis, placeat sequi
+            nam asperiores cupiditate beatae quod laborum non cum similique.
+          </p>
+        </div>
+        <div>
+          <h2 class="Social1">Our Social Media Outlets</h2>
+        </div>
+        <div class="Social2">
+          <a class="Facebook" href="www.facebook.com">Facebook</a>
+          <a class="Instagram" href="www.instagram.com">Instagram</a>
+          <a class="Reddit" href="www.reddit.com">Reddit</a>
```

Comment:
Make one of the links point to one of your HTML pages. 
`<a class="hood" target="_blank" href="./CLKW208HoodRichBulletproofEdition.html">Hood</a>`